src/main/java/com/robertx22/mine_and_slash/capability/entity/SummonedData.java

```diff
@@ -0,0 +1,19 @@
+package com.robertx22.mine_and_slash.capability.entity;
+
+import java.util.HashMap;
+
+public class SummonedData {
+    private HashMap<String, Integer> summonedTypes = new HashMap<>();
```

Comment:
Should I care about removing spells from this map when the player unlearns the spell?
It was not an easy task to inject into SpellSchoolsData so I am not sure if we even need such optimization